src/bo4e/com/adresse.py

```diff
@@ -5,20 +5,31 @@
 from bo4e.enum.landescode import Landescode
 
 
+def strasse_xor_postfach(instance, attribute, value):
+    if (
+        instance.strasse is None or instance.hausnummer is None
+    ) and instance.postfach is None:
```

Comment:
```suggestion
        not instance.strasse or not instance.hausnummer) and not instance.postfach:
```
ich frage mich, ob es hier nicht genug oder sogar angebrachter wäre, zu prüfen ob das Feld "truethy" oder "falsy" ist. Z.B. ein leerer String `strasse = ""` wäre ja ähnlich falsch würde aber durchgehen.

Reply:
Na klar, viel besser. Wussten nicht, dass das so einfach ist ^^